bin/citrea/src/lib.rs

```diff
@@ -16,10 +17,24 @@ mod bitcoin_rollup;
 pub use bitcoin_rollup::*;
 
 /// Default initialization of logging
-pub fn initialize_logging() {
+pub fn initialize_logging(level: Level) {
```

Comment:
i don't think it makes sense to have our main log initializer have a paremeter *just for* mock da tracing levels.

why not make a secondary init log function just for tests?